test/test_coverart_utils.py

```diff
@@ -3,14 +3,15 @@
 import os.path
 import shutil
 import tempfile
-import unittest
+from test.picardtestcase import PicardTestCase
 
 from picard.coverart.utils import translate_caa_type
 from picard.i18n import setup_gettext
 
 
-class CaaTypeTranslationTest(unittest.TestCase):
+class CaaTypeTranslationTest(PicardTestCase):
     def setUp(self):
+        super(CaaTypeTranslationTest, self).setUp()
```

Comment:
Why py2 style supers?

Reply:
Agreed, those should be fixed before merging